frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -732,6 +732,20 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).default,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      perspective: 'dev',
+      exact: false,
+      path: ['/k8s/all-namespaces/:plural'],
+      loader: async () =>
+        (
+          await import(
+            './components/ProjectSelectPage' /* webpackChunkName: "dev-console-buildconfigs" */
```

Comment:
update webpack chunk name.
```suggestion
            './components/ProjectSelectPage' /* webpackChunkName: "dev-console-projectselectpage" */
```